Gemfile.lock

```diff
@@ -700,7 +700,7 @@ DEPENDENCIES
   xmpp4r (~> 0.5.6)
 
 RUBY VERSION
-   ruby 2.3.1p112
+   ruby 2.2.2p95
```

Comment:
I will. having hard drive issues and it is getting ready to die. so I had to push before I lost it.